maintainers/scripts/luarocks-packages.csv

```diff
@@ -101,10 +101,10 @@ luaunbound,,,,,,
 luaunit,,,,,,lockejan
 luautf8,,,,,,pstn
 luazip,,,,,,
+lusc_luv,,,,,,
 lush.nvim,,,https://luarocks.org/dev,,,teto
```

Comment:
did the updater reorder the lines itself or have you done it manually (just curious :) ) ?

Reply:
I did, this is a manually edited file